README.rst

```diff
@@ -69,6 +77,8 @@ Copyright & Licensing
 All content © 2015 The Authors.
```

Comment:
Should this copyright year be updated to the current year? :D

Reply:
Ooops, done!